base/base_tests/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(base_tests C CXX)
+set(PROJECT_SOURCE_DIR ../..)
+add_definitions(-DOMIM_UNIT_TEST_DISABLE_PLATFORM_INIT)
+
+set(
+    SOURCES
+    assert_test.cpp
+    bits_test.cpp
+    buffer_vector_test.cpp
+    cache_test.cpp
+    collection_cast_test.cpp
+    condition_test.cpp
+    const_helper.cpp
+    containers_test.cpp
+    logging_test.cpp
+    math_test.cpp
+    matrix_test.cpp
+    mem_trie_test.cpp
+    newtype_test.cpp
+    observer_list_test.cpp
+    range_iterator_test.cpp
+    ref_counted_tests.cpp
+    regexp_test.cpp
+    rolling_hash_test.cpp
+    scope_guard_test.cpp
+    stl_add_test.cpp
+    stl_helpers_test.cpp
+    string_format_test.cpp
+    string_utils_test.cpp
+    sunrise_sunset_test.cpp
+    thread_pool_tests.cpp
+    threaded_list_test.cpp
+    threads_test.cpp
+    timegm_test.cpp
+    timer_test.cpp
+    worker_thread_test.cpp
+    ${PROJECT_SOURCE_DIR}/testing/testingmain.cpp
+)
+
+link_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+add_executable(base_tests ${SOURCES})
```

Comment:
Please, create a helper function like omim_add_executable, that'll link the library with threads by default.